portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,27 +15,43 @@
 package com.google.sps.servlets;
 
 import com.google.gson.Gson;
-import java.util.Arrays;
+import java.util.ArrayList;
 import java.util.List;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns a sample list of strings. */
-@WebServlet("/sample_strings")
+/** Servlet that receives user's comment (if any) and returns all users' comments for that session. */
+@WebServlet("/submit_comment")
```

Comment:
The `get` request doesn't seem to be `submit_comment`? Likely a naming issue.

Reply:
Sorry, but could you elaborate on this? I don't get it.😅

Reply:
I mean this `DataServlet` (which isn't a good name actually) has URL pattern `submit_comment`, however, this servlet is not just for submit comment, only its post request is for submit comment.

Reply:
For example, looking at these code
```
fetch('/submit_comment').then(response => response.json()).then(comments => {
    comments.forEach(commentText => {
      commentPlaceholder.appendChild(createCommentElement(commentText))
    });
  });
```
What response should we expect when we `submit_comment`, and what are we submitting here?

Reply:
I see. Yes, I will change the name of the class later (in another pull request). 

I agree that the URL is kind of confusing. I'll change it to `/comment` then. 

The get request should return all the comments for that session. 

Thanks!